### ISSUE TITLE
Unhandled Exception: NoSuchMethodError [trailerVideoId] on video

### ISSUE THREAD

**Issue by KarlaAngulo** [Hexer10/youtube_explode_dart#22]
when using
MediaStreamInfoSet stream = await yt.getVideoMediaStream("3qe8yBHkPpQ");

it returns me

Unhandled Exception: NoSuchMethodError: The method '[]' was called on null.
E/flutter (15487): Receiver: null
E/flutter (15487): Tried calling: []("trailerVideoId")

Why is this error? 
I also tried these videos and got the same error

https://www.youtube.com/watch?v=sIJfYix8RUI

**Comment by KarlaAngulo** [comment]
New error found with other music
https://www.youtube.com/watch?v=JRfuAukYTKg

Unhandled Exception: NoSuchMethodError: The method 'split' was called on null.
E/flutter (20143): Receiver: null
E/flutter (20143): Tried calling: split("&")

this returns me

**Comment by Tizoner27** [comment]
I've got the same error with
[https://youtu.be/e-xToC9wNl0](url)

```
The method 'split' was called on null.
Receiver: null
Tried calling: split("&")
dart:core                                                                                                                       Uri.splitQueryString
package:youtube_explode_dart/src/cipher/cipher.dart 100:23                                                                      decipherUrl
package:youtube_explode_dart/src/youtube_explode_base.dart 62:23 
```

**Comment by Hexer10** [comment]
Fixed with #23 open a new issue if this still persist.
